### ISSUE TITLE
The slider doesn't work with slides generated with a dom-repeat

### ISSUE THREAD

**Issue by mrpharderwijk** [ruyadorno/polymer-simple-slider#5]
I have the following code:

```html
<div id="simple-slider-container">
        <span id="prev-button" onclick="javascript:document.getElementById('simple-slider').prev()" href="#" class="slide-button-container">
          <iron-icon icon="hardware:keyboard-arrow-left" class="slide-button"></iron-icon>
        </span>
        <simple-slider id="simple-slider" transition-property="opacity" start-value="0" visible-value="1" end-value="0">
          
          <template
              is="dom-repeat" 
              items="[[ component.slides ]]">

             <img src="item.src" alt="item.alt" />

          </template>

        </simple-slider>
        <span id="next-button" onclick="javascript:document.getElementById('simple-slider').next()" href="#" class="slide-button-container">
          <iron-icon icon="hardware:keyboard-arrow-right" class="slide-button"></iron-icon>
        </span>
      </div>
```

The code doesn't seem to handle the dom-repeat well, because it places the slide css on the template dom-repeat tag. Please advice!

**Comment by ruyadorno** [comment]
hi @mrpharderwijk thanks for the report, I took 5min yesterday to try and replicate the problem yesterday  but I couldn't get a minimal environment working within that time...

could you help me out by providing a minimal setup to try and debug it, that would be really appreciated 😊 

maybe just generate a new project using `generator-polymer` and push all the files to a github repo so that I can clone and give it a try

Thanks again!

**Comment by mrpharderwijk** [comment]
Sorry for my late response, let me get back to you on that today. I'm creating a repo as we speak.

**Comment by mrpharderwijk** [comment]
Dear @ruyadorno,

Please advice on below repo:
https://github.com/mrpharderwijk/simple-slider-issue-5

**Comment by ruyadorno** [comment]
Alright, just validated @TiemeTogola PR within your example and it fixes the problem 😊 I just made a new [v1.1.0 release](https://github.com/ruyadorno/polymer-simple-slider/releases/tag/v1.1.0) containing that!

Mind that you'll need to add an `auto-reload` attribute to your element declaration.

Many thanks @mrpharderwijk for providing the report along with such a great example! 💪 And thanks again @TiemeTogola for providing a timely fix ⚡️

**Comment by mrpharderwijk** [comment]
Nice, I will make sure to test and implement it in the application of my current client. Since the prio of the ticket is in the current sprint this might be this week 😉 ... Let's keep our 🤞
